tests/data/preview_py_310/pep604_union_types_line_breaks.py

```diff
@@ -172,16 +218,100 @@ def foo(
 
 @app.get("/path/")
 async def foo(
-    q: str | None = Query(
-        None, title="Some long title", description="Some long description"
-    )
+    q: str
+        | None
+        = Query(None, title="Some long title", description="Some long description")
 ):
     pass
 
 
 def f(
-    max_jobs: int | None = Option(
-        None, help="Maximum number of jobs to launch. And some additional text."
-    ),
+    max_jobs: int
+        | None
+        = Option(
+            None, help="Maximum number of jobs to launch. And some additional text."
+        ),
     another_option: bool = False,
 ): ...
+
+
+# don't lose comments
+def f(  # 1
+    looooooooooooooooooooooooooong:  # 2  # 3
+        Loooooooooooooooooooooooooooooooooooooooooooooooong  # 4
+        | Loooooooooooooooooooooooooooooooooooooooooooooooong  # 5  # 6
+        = 3,  # 7  # 8
+):  # 9
+    ...  # 10
+
+
+loooooooooooooooooooooooooooooong: (
+    Loooooooooooooooooooooooooooong  # aoeuaoeuaoeuaoeuaoeuaoeu
+)
+
+
+def foo(
+    loooooooooooooooooooooooooooooooooooooooooooooooooooooooooooong:
+        Loooooooooooooooooooooooooooong,
+): ...
+
+
+def foo(
+    loooooooooooooooong: Tuple[
+        Looooooooooooooooooooooooooooong, Loooooooooooooong
+    ],  # aoeuaoeuaoeu
+): ...
+
+
+def foo(
+    a: loooooooooooooong
+        | loooooooooooooooooooooooooooong
+        | looooooooooong
+        | looooooooooong
+        = 3,
```

Comment:
All these examples with defaults look bad. It's as if we're doing long division.